ChangeLog.md

```diff
@@ -52,6 +52,9 @@ See docs/process.md for more on how version tagging works.
   impossible to wait for the function to actually finish and retrieve its
   result. Now in those cases it will return a `Promise` instead that will
   resolve with the function's return value upon completion. (#11890)
+- Added `EM_ASYNC_JS` macro - similar to `EM_JS`, but allows using `await`
+  inside the JS block and automatically integrates with Asyncify without
+  the need for listing the declared function in `ASYNCIFY_IMPORTS` (#9709).
```

Comment:
This changelog note should probably be moved to 2.0.27 (since 2.0.26 was tagged before this PR landed).

Reply:
Hmm, was it? At least the release PR came after but I'm not sure about tag itself: https://github.com/emscripten-core/emscripten/commits/main

Reply:
Oh, I see, it does look like tag came in earlier.

Reply:
https://github.com/emscripten-core/emscripten/pull/14765